packages/components/src/internal/components/entities/EntityInsertPanel.tsx

```diff
@@ -273,16 +273,18 @@ export class EntityInsertPanelImpl extends Component<Props, StateProps> {
 
         const allowParents = this.allowParents();
 
-        try {
-            const nameIdSettings = await this.props.loadNameExpressionOptions();
-            this.setState({ allowUserSpecifiedNames: nameIdSettings.allowUserSpecifiedNames });
-        } catch (error) {
-            this.setState({
-                error: getActionErrorMessage(
-                    'There was a problem retrieving name expression options.',
-                    this.typeTextPlural
-                ),
-            });
+        if (isSampleManagerEnabled()) {
+            try {
+                const nameIdSettings = await this.props.loadNameExpressionOptions();
+                this.setState({ allowUserSpecifiedNames: nameIdSettings.allowUserSpecifiedNames });
+            } catch (error) {
+                this.setState({
+                    error: getActionErrorMessage(
+                        'There was a problem retrieving name expression options.',
+                        this.typeTextPlural
+                    ),
+                });
+            }
         }
```

Comment:
unrelated to this PR but I noticed in the error message for this component (at line 283) it refers to "name expression" but for the `SampleTypePropertiesPanel.tsx` it refers to "Naming Pattern". Should we be consistent in the text we show the user?

Reply:
There are a small handful of places where it's inconsistent—I think I'll do a sweep of the ones I can find, and open an issue for all of them soon.